packages/dev-tools/src/summary/AccountSummary.tsx

```diff
@@ -1,12 +1,15 @@
 import { useEffect, useState } from "react";
-import { formatUnits } from "viem";
+import { formatUnits, parseEther, testActions } from "viem";
 import { useDevToolsContext } from "../DevToolsContext";
 
 export function AccountSummary() {
   const { publicClient, walletClient } = useDevToolsContext();
+  const walletAddress = walletClient?.account?.address;
+  const testClient = publicClient.chain.id === 31337 ? publicClient.extend(testActions({ mode: "anvil" })) : null;
```

Comment:
deriving this from the public client for now because wallet client in account-kit branch gave errors due to it pointing at a bundler instance, which rejected the anvil rpc methods